jobs/transforms/blog_validation.py

```diff
@@ -65,24 +66,21 @@ def _get_model_domain_object_instance(self, blog_post_model):
             blog_post_model.published_on
         )
 
-    def _get_domain_object_validation_type(self, item):
+    def _get_domain_object_validation_type(self, unused_item):
         # type: (Any) -> Any
         """Returns the type of domain object validation to be performed.
 
         Args:
-            item: datastore_services.Model. Entity to validate.
+            unused_item: datastore_services.Model. Entity to validate.
 
         Returns:
             str. The type of validation mode: strict or non strict.
         """
-        blog_post_rights = (
-            blog_services.get_blog_post_rights( # type: ignore[no-untyped-call]
-                item.id, strict=True))
-
-        if blog_post_rights.blog_post_is_published:
-            return base_validation.VALIDATION_MODES.strict
-
-        return base_validation.VALIDATION_MODES.non_strict
+        # TODO(#13397): Write a custom job to avoid applying strict validation
```

Comment:
Could you please follow up with @iamprayush too (Rijuta's mentor) to make sure he is aware of this?

Reply:
Yep, done.